open-api/lang-parser/src/main/java/io/deephaven/lang/generated/ChunkerMixin.java

```diff
@@ -436,6 +520,9 @@ public Checker is(String s) {
                 private int pntr;
                 @Override
                 public int check(char c) {
+//                    if (s.length() <= pntr){
```

Comment:
remove or todo?

Reply:
uncomment, it's better to fail normally than to get an IOOBE later.